rmw_fastrtps_shared_cpp/QUALITY_DECLARATION.md

```diff
@@ -85,7 +85,7 @@ The results of the test can be found [here](https://ci.ros2.org/view/nightly/job
 
 ### Feature Testing [4.i]
 
-Many of the features have testing in `rmw_fastrtps_shared_cpp`.
+Many of the features have testing in `rmw_fastrtps_shared_cpp`. Since `rmw_fastrtps_shared_cpp` supports `rmw_fastrtps_cpp` and `rmw_fastrtps_dynamic_cpp`, integration and system tests for ROS middlewares up the stack also provide coverage.
```

Comment:
Same comment.

Reply:
See https://github.com/ros2/rmw_fastrtps/pull/380/commits/4e7316ad6346f9e7a7253ae1ee81961ac346092c. I refrained from mentioning CI builds, as `rmw_fastrtps_shared_cpp` isn't an actual implementation and it already mentions `rmw_fastrtps_cpp`.